acsylla/_cython/cpp_cassandra.pxi

```diff
@@ -175,17 +186,24 @@ cdef extern from "cassandra.h":
   CassError cass_value_get_float(const CassValue* value, cass_float_t* output)
   CassError cass_value_get_bool(const CassValue* value, cass_bool_t* output)
   CassError cass_value_get_string(const CassValue* value, const char** output, size_t* output_size)
-  CassError cass_value_get_bytes(const CassValue* value, const cass_byte_t** output, size_t* output_size);
-  CassError cass_value_get_bytes(const CassValue* value, const cass_byte_t** output, size_t* output_size);
-
+  CassError cass_value_get_bytes(const CassValue* value, const cass_byte_t** output, size_t* output_size)
+  CassError cass_value_get_bytes(const CassValue* value, const cass_byte_t** output, size_t* output_size)
+  CassError cass_value_get_uuid(const CassValue* value, CassUuid* output)
   CassRow* cass_iterator_get_row(const CassIterator* iterator)
   cass_bool_t cass_iterator_next(CassIterator* iterator)
   void cass_iterator_free(CassIterator* iterator)
 
-  CassStatement* cass_prepared_bind(const CassPrepared* prepared);
-  void cass_prepared_free(const CassPrepared* prepared);
+  CassStatement* cass_prepared_bind(const CassPrepared* prepared)
+  void cass_prepared_free(const CassPrepared* prepared)
 
 
   CassBatch* cass_batch_new(CassBatchType type)
   CassError cass_batch_add_statement(CassBatch* batch, CassStatement* statement)
   void cass_batch_free(CassBatch* cass_batch)
+
+  CassError cass_uuid_from_string(const char* str, CassUuid* output)
+  void cass_uuid_string(CassUuid uuid, char* output)
+  cass_uint64_t cass_uuid_timestamp(CassUuid uuid)
+
+  void cass_uuid_min_from_time(cass_uint64_t time, CassUuid* output)
+  void cass_uuid_max_from_time(cass_uint64_t time, CassUuid* output)
```

Comment:
Rember also to remove these references also, they shouldn't be used now

Reply:
thanks! done.